packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -166,7 +192,9 @@ export class Copy {
             fs.rmSync(downloadDir, {recursive: true});
             return {
                 success:true,
-                commandResponse: ZosFilesMessages.datasetCopiedSuccessfully.message
+                commandResponse: newDataSet
+                    ? util.format(ZosFilesMessages.newDataSetCreated.message, toPds)
```

Comment:
Would it make sense to instead have the calling function change the command response message if it created a new data set? I just ask because this function isn't creating the new data set itself.